README.md

```diff
@@ -19,6 +19,43 @@
 ### Configuration
```

Comment:
Add #### Network ports
Now that we have #### External config files
That makes it easier to read as a whole.

Reply:
Changed